testing_exercise/testingSpec.js

```diff
@@ -1,3 +1,49 @@
 var expect = chai.expect;
 
-// WRITE YOUR TESTS HERE!
\ No newline at end of file
+// WRITE YOUR TESTS HERE!
+describe("Tests", function(){
+
+	it("replaces lowercase", function(){
+		expect(replaceWith("awesome", "e", "z")).to.be.equal("awzsomz");
+	});
+
+	it("replaces case sensitive", function(){
+		expect(replaceWith("Foo", "F", "B")).to.be.equal("Boo");
+	});
+
+	// unclear test case:
+	// replaceWith("Woo", "W", "p"); // Poo? or poo? thus far unspecified.
+
+	it("Expand: ", function(){
+		expect(expand([1,2,3],3)).to.be.deep.equal([1,2,3,1,2,3,1,2,3]);
+		expect(expand(["foo", "test"],1)).to.be.deep.equal(["foo","test"]);
+	});
+
+	it("Does it accept only numbers?", function(){
+		expect(acceptsOnlyNumbers(1,"foo")).to.be.equal(false);
+		expect(acceptsOnlyNumbers(1,2,3,4,5,6,7)).to.be.equal(true);
+		expect(acceptsOnlyNumbers(1,2,3,4,5,6,NaN)).to.be.equal(false);
+	});
+
+	it("mergeArrays", function(){
```

Comment:
In general, I'd try to make the strings inside of your `it` blocks more descriptive. What exactly about these functions are you testing?